src/modules/auth/auth.controller.ts

```diff
@@ -18,4 +19,16 @@ export class AuthController {
   register(@Body() registerDto: RegisterDto) {
     return this.authService.register(registerDto);
   }
+
+  @Post('login')
+  @HttpCode(200)
```

Comment:
I don't think you have to set this explicitly, because 200 is already the default code

Reply:
Default was 201